docs/output.md

```diff
@@ -201,43 +204,56 @@ Files in these two folders contain all contigs of an assembly.
+* `quast_and_busco_summary.tsv`: Summary of BUSCO and QUAST results
 
 ### QC for metagenome assembled genomes with BUSCO
 
 [BUSCO](https://busco-archive.ezlab.org/v3/) is a tool used to assess the completeness of a genome assembly. It is run on all the genome bins and high quality contigs obtained by MetaBAT2.
 
-**Output directory: `results/GenomeBinning/QC/BUSCO`**
+**Output directory: `GenomeBinning/QC/BUSCO`**
 
-- `[assembler]-[bin]_busco_log.txt`: BUSCO log file
-- `[assembler]-[bin]_busco.fna`: Nucleotide sequence of all identified BUSCOs
-- `[assembler]-[bin]_busco.faa`: Aminoacid sequence of all identified BUSCOs
+* `[assembler]-[bin]_busco_log.txt`: BUSCO log file
+* `[assembler]-[bin]_busco.fna`: Nucleotide sequence of all identified BUSCOs
+* `[assembler]-[bin]_busco.faa`: Aminoacid sequence of all identified BUSCOs
 
-**Output directory: `results/GenomeBinning/QC`**
+**Output directory: `GenomeBinning/QC`**
 
-- `busco_summary.txt`: A summary table of the BUSCO results, with % of marker genes found
-- `quast_and_busco_summary.tsv`; Summary of BUSCO and QUAST results
+* `busco_summary.txt`: A summary table of the BUSCO results, with % of marker genes found
+* `quast_and_busco_summary.tsv`; Summary of BUSCO and QUAST results
 
 ## MultiQC
 
-[MultiQC](http://multiqc.info) is a visualisation tool that generates a single HTML report summarising all samples in your project. Most of the pipeline QC results are visualised in the report and further statistics are available in within the report data directory.
+[MultiQC](http://multiqc.info) is a visualization tool that generates a single HTML report summarizing all samples in your project. Most of the pipeline QC results are visualised in the report and further statistics are available in the report data directory.
+
+The pipeline has special steps which also allow the software versions to be reported in the MultiQC output for future traceability.
+
+For more information about how to use MultiQC reports, see [https://multiqc.info](https://multiqc.info).
+
+**Output files:**
+
+* `multiqc/`
+  * `multiqc_report.html`: a standalone HTML file that can be viewed in your web browser.
+  * `multiqc_data/`: directory containing parsed statistics from the different tools used in the pipeline.
+  * `multiqc_plots/`: directory containing static images from the report in various formats.
+
+## Pipeline information
 
-The pipeline has special steps which allow the software versions used to be reported in the MultiQC output for future traceability.
+[Nextflow](https://www.nextflow.io/docs/latest/tracing.html) provides excellent functionality for generating various reports relevant to the running and execution of the pipeline. This will allow you to troubleshoot errors with the running of the pipeline, and also provide you with other information such as launch commands, run times and resource usage.
 
-**Output directory: `results/multiqc`**
+**Output directory: `results/pipeline_info`**
```

Comment:
You always removed "results/" above, but here it remained. Might this occurrence be better removed as well?